plugins/BEdita/Core/config/Migrations/20180614080050_AddTranslationsTable.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2018 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+use Migrations\AbstractMigration;
+
+/**
+ * Migration class to create `translations` table.
+ */
+class AddTranslationsTable extends AbstractMigration
+{
+
+    /**
+     * {@inheritDoc}
+     */
+    public $autoId = false;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function up()
+    {
+        $columnTypes = $this->getAdapter()->getColumnTypes();
+        $enum = in_array('enum', $columnTypes) ? 'enum' : 'string';
+
+        $this->table('translations')
+            ->addColumn('id', 'integer', [
+                'autoIncrement' => true,
+                'default' => null,
+                'limit' => 10,
+                'null' => false,
+                'signed' => false,
+            ])
+            ->addPrimaryKey(['id'])
+            ->addColumn('object_id', 'integer', [
+                'comment' => 'link to annotated object',
```

Comment:
[minor] `link to translated object` maybe?

Reply:
Right. Leftover from copy-paste.